public/js/profile.js

```diff
@@ -120,9 +120,15 @@ if (logout_user) {
 
 
 function logoutUSer() {
-    prompt('Are you sure? ...... (Yes/No)')
-    localStorage.removeItem('token');
-    window.location.replace('/login');
+   let response =  prompt('Are you sure? ...... (Yes/No)')
+    if (response != null) {
+        if (response == "yes" || response == "Yes") {
+            localStorage.removeItem('token');
+            window.location.replace('/login'); 
+        }
+        
+    }
```

Comment:
Why are we using the prompt message in the first place, the user can always login if he needs to make use of the account.

Please @odomogochuwu please change this to enable Toast notification to users that they are logged out